external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -114,7 +114,7 @@
           Condition="'$(TargetGroup)'=='uap'" >
 
     <PropertyGroup>
-      <UAPToolsPackageVersion>1.0.2</UAPToolsPackageVersion>
+      <UAPToolsPackageVersion>1.0.4</UAPToolsPackageVersion>
```

Comment:
yup, we already talked about this, we will have to create a new version entirely for his changes to work, so we will rebase this PR once mine is merged.